src/test/scala/com/fulcrumgenomics/umi/ExtractUmisFromBamTest.scala

```diff
@@ -38,6 +38,11 @@ class ExtractUmisFromBamTest extends UnitSpec with OptionValues {
 
   def newBam = makeTempFile("extract_umis_from_bam_test.", ".bam")
 
+  /** An implicit conversion to read stucture to make test writing easier below by allowing the use of Strings
+    * directly instead of having to write ReadStructure("75T") everywhere.
+    */
+  private implicit def stringToReadStructure(rs: String): ReadStructure = ReadStructure(rs)
```

Comment:
I'm really pleased I thought of this instead of rewriting all the lines in the test that construct `ExtractReadsFromUmi` to have e.g. `readStructures=Seq(ReadStructure("75T"))` instead of `readStructures=Seq("75T")`.  This makes me thing that we should actually make _more_ use of private implicits in our tests to make things easier to read.